### ISSUE TITLE
[AUTOMATION] Load Facility Health Services data [Engineer]

### ISSUE THREAD

**Issue by SgardnerPMP** [department-of-veterans-affairs/va.gov-team#10097]
## User Story or Problem Statement
As a Data Tech, I need an optimized way to set content into existing content fields from a .csv or .txt file, so I can reduce time in loading and completing page content in VAMC- Web page/nodes.

## Goal
_Implement required va.gov webpages_

## Objectives or Key Results this is meant to further
- _Increase pages/nodes completed in a quarter_

## Resources - Tools - Documentation
[Review System and facility spreadsheet](https://dvagov.sharepoint.com/:f:/s/VHADigitalMedia/vamc-upgrade-project/Ej33rnb0EAZDkvT2A-696qUBx40SHv8dOusH-nP810orTg?e=3JLBLQ)

## Tasks
_Creation of the following Drupal Facility Level Health Services_
- [ ] Create Facility health service script to migrate .csv/script 
- [ ] Confirm .csv script is correct for migration in local environment
- [ ] Create script and submit Push Request for testing in Devshop
- [ ] Include the following acceptance criteria within PR
* Review facility health services page 
* Confirm Content in .csv was loaded to correct System
* Confirm formatting is present in facility health services blobby (Text Area)
- [ ] If there is feedback or script issues:
* Communicate in PR that you understand issues and are working them immediately
* Update script with resolutions to feedback and requests from PR
- [ ] Submit new Commit to PR with revisions 
- [ ] CMS must provide approval and they will Merge into Devshop.
- [ ] CMS must merge PR with master (Assure this is communicated)
- [ ] Once Merge is complete, check Prod.cms.va.gov to validate data has been loaded.  

## Acceptance Criteria
_Validate outcome of pushing content to existing nodes_
* PR submitted for migration of .csv data into Facility Health Services - "Wilks-Barre"
- Review facility health services page 
- Confirm Content in .csv was loaded to correct System
- Confirm formatting is present in facility health services blobby (Text Area)
* PR has been approved and merged with Master by CMS
* Data is confirmed to be loaded in prod.cms.va.gov
